src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,14 @@
 // Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type TodosProps = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<TodosProps> = ({ todos }) => (
+  <ul className="TodoList">
+    {todos.map((todo) => (<TodoInfo todo={todo} key={todo.id} />))}
```

Comment:
I understand the temptations to provide one-line solution 😄 
But clearness of the code is our top priority
```suggestion
    {todos.map((todo) => (
      <TodoInfo todo={todo} key={todo.id} />
    ))}
```